src/py42/sdk/queries/__init__.py

```diff
@@ -9,6 +14,13 @@ def __init__(self, *args, **kwargs):
         self.page_number = None
         self.sort_key = None
 
+    @classmethod
```

Comment:
added `from_dict` to `BaseQuery`, `FilterGroup`, and `QueryFilter`. This allows these objects to be more easily built directly from JSON responses like those that come from saved search.